components/booking/overview/AccessibilityOverview.vue

```diff
@@ -0,0 +1,105 @@
+<!-- eslint-disable vue/no-mutating-props -->
+<template>
+  <overview-box>
+    <template #title>
+      <font-awesome-icon icon="info" class="mr-2" />
+      Accessibility Info
+    </template>
+    <div class="p-2 px-4 bg-sta-gray rounded">
+      <template v-if="!changingAccessibility">
+        <div class="flex">
+          <p class="flex-grow py-2">
+            {{ booking.accessibilityInfo }}
+          </p>
+          <UiStaButton
+            class="bg-sta-orange hover:bg-sta-orange-dark transition-colors lg:mx-8 mx-4"
+            @click="
+              () => {
+                changingAccessibility = true;
+                newAccessibility = booking.accessibilityInfo;
+              }
+            "
+          >
+            Edit
+          </UiStaButton>
+        </div>
+      </template>
+      <template v-else>
+        <div class="flex">
+          <form-label class="flex-grow">
+            <UiInputText
+              v-model="newAccessibility"
+              placeholder="e.g. Wheelchair seat required"
+              :errors="errors"
+              required
+            />
+          </form-label>
+          <UiStaButton
+            class="bg-sta-green hover:bg-sta-green-dark transition-colors lg:mx-8 mx-4"
+            @click="
+              () => {
+                booking.accessibilityInfo = newAccessibility;
+                updateAPI();
```

Comment:
Just double-checking it is the case on the back-end too; will the prod team be emailed about an update made to accessibility info after the booking has been already confirmed?

Reply:
This is a great catch, and led me to a further investigative chain. I've pulled the PR back to draft, since it's going to need some further API work to go with it to make sure that emails are sent properly.